proposed/bandwidth-considerations.md

```diff
@@ -0,0 +1,80 @@
+_Note: This recommendation is a DRAFT for consideration of the MODX Governance Advisory Board._
+
+
+To be mindful of the cost users pay to access the MODX Next management experience the core shall effectively leverage the browser cache.
+
+
+**Editor: JP de Vries**
+**First publish draft: TBD**
+
+
+## Goals of Recommendation
+
+
+To make the MODX Next management experience more accessible and less expensive for our global user base. To optimize the page load times of the MODX Next manager across the web as a whole. To promote stability of the MODX Next manager experience. To add costly features and enhancements in a prgoressive manor so they can be opted out&nbsp;of.
+
+
+## Relevant Recommendations
+As bandwidth considerations is part of accessibility best practices, [DRAFT] Accessibility Mandate. This recommendation is proposed as a guideline accepted by the MODX Advisory Board and enforced by the Accessibility Mandate.
+
+
+## Recommendation
+Leverage the browser cache and stability of the progressively enhanced manager with the following practices:
+
+
+Front End (CSS, JS) assets delivered with the core shall not be compressed or concatenated on the server side.
+Front End assets delivered with the core shall first perform feature detection to test whether or not they are supported and not be loaded if they do not support the user’s environment.
+Common Dependencies used by the core such as jQuery, React, AngularJS, etc shall conditionally be loaded from a common CDN with local fallbacks to preserve stability in the event the CDN is unreachable.
+Extra Developers shall be encouraged, but not required, to perform feature detection to determine if a given asset has already been loaded.
+Server Admins will be encouraged to optimize the delivery of MODX by enabling HTTPS (HTTP/2), GZIP compression, and far future reaching expiry dates
+
+
+A great deal of research, testing, and debate went into the changes introduced in 2.5 that improved the stability and performance of the delivery of front end dependencies. The TL;DR is compressing and concatenating scripts on the server side is an anti optimization. This worst practice made the Manager experience less stable, as it was the cause of dozens of “white screen of death” issues.
+
+
+The MODX Manager is not a website! The assets it loads vary from page to page, user to user, and change as Extras are enabled, installed, and disabled. From a bandwidth perspective, the Manager is the opposite of a web site. As proven in the Proof of Concept section, optimization that commonly apply to websites are anti-optimal when applied to the manager experience.
+
+
+
+
+## Proof of Concept
```

Comment:
Further to another comment, this doesn't necessarily recommend something new. It's worthwhile research to prove a point that could be linked to, but I'm not sure if this should be in the actual recommendation. Perhaps a simple proof of concept github repo people can interact with instead?

One reason I'd personally challenge putting too much original "research" into a recommendation, is that the research may change over time. Better examples, a live concept someone can actually try out, or perhaps something new comes out that changes the research. A recommendation will be pretty much static once accepted.